src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] I think it would be better to write unit tests with [React Testing Library](https://www.freecodecamp.org/news/write-unit-tests-using-react-testing-library/) and Jest (for unit tests) for all your React app. currently, it has one failing test 😎😎

   ![image](https://user-images.githubusercontent.com/102757126/216518645-d73c4195-d73b-4e0e-9094-40387a1f7e96.png)


  **Remember**,[ Unit tests ](https://www.freecodecamp.org/news/write-unit-tests-using-react-testing-library/)are an important part of software development, as they help ensure that individual units of code (such as React components) work as intended, and continue to work as intended after changes are made. 👍 👍